pom.xml

```diff
@@ -240,6 +241,11 @@
                 <artifactId>jackson-dataformat-cbor</artifactId>
                 <version>${jackson.cbor.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-databind</artifactId>
+                <version>${jackson.databind.version}</version>
```

Comment:
I'd missed this during the original changes, sorry about that. We really shouldn't be adding individual jackson modules to the common pom. The whole point of using the bom is to only have [that one place](https://github.com/FasterXML/jackson-bom/blob/master/pom.xml#L90) 
 where these are defined. Otherwise all future updates need to update both the bom and these.

We should just remove all version properties and dependencies other then the bom version.